client/src/app/models/site.coffee

```diff
@@ -0,0 +1,10 @@
+Backbone = require('backbone')
+
+module.exports = Backbone.Model.extend(
+  urlRoot: 'http://localhost:8989/sites'
```

Comment:
Since the front end port is different (1337), will it still work in development? That was mostly for dev purposes.